app/build.gradle

```diff
@@ -47,6 +47,7 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.7.0'
     implementation 'androidx.appcompat:appcompat:1.4.2'
     implementation 'com.google.android.material:material:1.6.1'
+    implementation 'androidx.compose.ui:ui-graphics:1.1.1'
```

Comment:
Added compose UI graphics in build.gradle as an attempt to use Color() was made in colors (original code). https://developer.android.com/jetpack/androidx/releases/compose-ui